wiki/_styles/style.css

```diff
@@ -4,7 +4,7 @@
 	--bg-02: hsl(0, 0%, 19%);
 	--bg-03: hsl(0, 0%, 25%);
 	--bg-04: hsl(0, 0%, 31%);
-	--bg-05: hsl(0, 0%, 45%);
+	--bg-05: hsl(0, 0%, 35%);
```

Comment:
Create a new `--bg-06` instead. We don't know what else references the existing colour.